spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/repository/query/SpannerStatementQueryExecutor.java

```diff
@@ -73,6 +74,37 @@
 				sqlAndTags.getFirst(), sqlAndTags.getSecond(), null, params));
 	}
 
+	/**
+	 * Gets a query that returns the rows associated with a parent entity. This function
+	 * is intended to be used with parent-child interleaved tables, so that the retrieval
+	 * of all child rows having the parent's key values is efficient.
+	 * @param parentPersistentEntity the persistent entity of the parent table.
+	 * @param parentObject the parent object whose children to get.
+	 * @param childTable the name of the child table.
+	 * @return the Spanner statement to perform the retrieval.
+	 */
+	public static Statement getChildrenRowsQuery(
+			SpannerPersistentEntity parentPersistentEntity, Object parentObject,
+			String childTable) {
```

Comment:
Add child persistent entity to mention only those columns from that metadata in this select statement